src/tests/JIT/Regression/JitBlue/Runtime_105719/Runtime_105719.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- Needed for CLRTestEnvironmentVariable -->
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
+    <Optimize>True</Optimize>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildProjectName).cs" />
+
+    <CLRTestEnvironmentVariable Include="DOTNET_TieredCompilation" Value="0" />
+  </ItemGroup>
+</Project>
```

Comment:
This can use a [simpler](https://github.com/dotnet/runtime/blob/main/src/tests/JIT/Regression/JitBlue/Runtime_105742/Runtime_105742.csproj) template as well (note that we run all testing with and without tiered compilation enabled).

Reply:
Done!